code/game/turfs/simulated/floor_types.dm

```diff
@@ -121,6 +121,26 @@
 			F.make_plating()
 			return
 
+/turf/simulated/floor/engine/ex_act(severity)
+	switch(severity)
+		if(1.0)
+			if(prob(80))
+				src.ReplaceWithLattice()
+			else if(prob(50))
+				src.ChangeTurf(under_turf)
+			else
+				var/turf/simulated/floor/F = src

```

Comment:
There probably was a reason when I first wrote that but I can't see it anymore myself.